src/cls.ts

```diff
@@ -16,47 +16,37 @@
 
 'use strict';
 
-var semver = require('semver');
-var cls = semver.satisfies(process.version, '>=8') &&
-          process.env.GCLOUD_TRACE_NEW_CONTEXT ?
-            require('./cls-ah'/*.js*/) : require('continuation-local-storage');
+import * as semver from 'semver';
+import * as CLS from 'continuation-local-storage';
 
-/** @const {string} */
-var TRACE_NAMESPACE = 'com.google.cloud.trace';
+const cls: typeof CLS = semver.satisfies(process.version, '>=8') &&
```

Comment:
Why is `typeof` used here?

Reply:
Since `tsc` can't infer the type of `cls` from the RHS, we can set the type to `typeof CLS`, which is the type of the object we get from importing from `continuation-local-storage`.